tests/document_stream_tests.cpp

```diff
@@ -69,6 +69,69 @@ namespace document_stream_tests {
     }
     return true;
   }
+  bool single_document() {
```

Comment:
Oh wow, we didn't have a single document test? That's crazy. Yay :)

Reply:
Yes. I was pretty sure that it would work.